### ISSUE TITLE
Uncaught DOMException: Failed to read the 'localStorage' property from 'Window': Access is denied for this document.

### ISSUE THREAD

**Issue by Liupeng8** [nklayman/vue-cli-plugin-electron-builder#265]
**Describe the bug**
Everything works just fine in development mode but I'm getting this error in the devtools console after building my app when i use the tinymce4.7.5 and cookie

`tinymce.min.js:2 Uncaught DOMException: Failed to read the 'localStorage' property from 'Window': Access is denied for this document.
    at app://./static/tinymce4.7.5/tinymce.min.js:2:334766
    at app://./static/tinymce4.7.5/tinymce.min.js:2:337378`

`Uncaught DOMException: Failed to read the 'cookie' property from 'Document': Access is denied for this document.
    at Function.i [as set] (app://./js/chunk-vendors.8d12f151.js:70:1078)
    at Function.i.get (app://./js/chunk-vendors.8d12f151.js:70:1494)
    at v (app://./js/app.58b4478d.js:1:68573)
    at Object.9923 (app://./js/app.58b4478d.js:1:68773)
    at r (app://./js/app.58b4478d.js:1:2352)
    at Module.d9cd (app://./js/app.58b4478d.js:1:122194)
    at r (app://./js/app.58b4478d.js:1:2352)
    at i (app://./js/app.58b4478d.js:1:120363)
    at app://./js/app.58b4478d.js:1:29578
    at Array.reduce (<anonymous>)`

**To Reproduce**
yarn electron:build
and then open the the app

**Expected behavior**
Get access to LocalStorage and Cookie

**Screenshots**
![123](https://user-images.githubusercontent.com/23046680/57294771-44824100-70fb-11e9-91d5-49b231cfece1.png)

**Environment (please complete the following information):**
OS and version: windows vs code
node version: 12.1.0
npm version: 6.9
yarn version: 1.10.1
vue-cli-plugin-electron-builder version : 1.3.1
electron version: 5.0.1

vue.config.js:
`'use strict'
const path = require('path')

function resolve(dir) {
  return path.join(__dirname, dir)
}

module.exports = {
  pluginOptions: {
    electronBuilder: {
      outputDir: 'build',
      externals: ['localStorage', 'cookie'],
      chainWebpackRendererProcess: config => {
        // set svg-sprite-loader
        config.module
          .rule('svg')
          .exclude.add(resolve('src/icons'))
          .end()
        config.module
          .rule('icons')
          .test(/\.svg$/)
          .include.add(resolve('src/icons'))
          .end()
          .use('svg-sprite-loader')
          .loader('svg-sprite-loader')
          .options({
            symbolId: 'icon-[name]'
          })
          .end()
      }
    }
  }
}`

**Additional context**
I tried to introduce tinymce in the basic framework, and the same error will occur.

**Comment by Liupeng8** [comment]
@nklayman If you are convenient, please help me to look at my problem.

**Comment by crossle** [comment]
https://github.com/nklayman/vue-cli-plugin-electron-builder/issues/262#issuecomment-490041507

**Comment by Liupeng8** [comment]
@crossle 兄台，再请教你一个问题，先谢谢了。
我的应用中使用了[https://github.com/zenorocha/clipboard.js](url)，开发模式下一切正常，但是编译以后，出现了下面的报错
![image](https://user-images.githubusercontent.com/23046680/57345864-13e3eb00-717f-11e9-8af8-e290abdc4ef6.png)

![image](https://user-images.githubusercontent.com/23046680/57345909-34ac4080-717f-11e9-89d3-71d136203be7.png)

刚开始用vue-cli-plugin-electron-builder这个框架，请多多指教

**Comment by nklayman** [comment]
These was caused by the `app://` scheme not being registered as standard. This is fixed as of version `1.3.2`. If you were previously using an older version, you must manually update your `background.js`:

```js
// Replace
protocol.registerSchemesAsPrivileged([{scheme: 'app', privileges: { secure: true } }])
// With
protocol.registerSchemesAsPrivileged([{scheme: 'app', privileges: { secure: true, standard: true } }])
```

**Comment by dortonway** [comment]
How to fix the error in usual electron.js?
